standalone/tsconfig.json

```diff
@@ -29,43 +26,21 @@
       "common/*": [
         "projects/common/*"
       ],
-      "facia/*": [
-        "projects/facia/*"
-      ],
-      "journalism/*": [
-        "projects/journalism/*"
-      ],
-      "lib/*": [
-        "lib/*"
-      ],
-      "lodash/*": [
-        "node_modules/lodash-node/compat/*"
-      ],
-      "membership/*": [
-        "projects/membership/*"
-      ],
-      "ophan/embed": [
-        "node_modules/ophan-tracker-js/build/ophan.embed"
-      ],
-      "ophan/ng": [
-        "node_modules/ophan-tracker-js"
-      ],
```

Comment:
Thanks for removing these paths!

I was just curious about this `ophan/ng` one - it looks like it's used in `ab-ophan.ts` which we'll copy over from Frontend, do we no longer need this?

Reply:
I think `ab-ophan.ts` is in because `ab.ts` which imports all the tests is needed for the ab testing to work.
We haven't considered how ab-testing will work have we 🤔

Reply:
That makes sense, I was just wondering if we needed to retain this `ophan/ng` path for that to work?

> We haven't considered how ab-testing will work have we 🤔

This lib will become another place we'll have to duplicate AB tests maybe?